src/spectrum.rs

```diff
@@ -0,0 +1,397 @@
+use std::fmt;
+use std::ops::{Add, Sub, Mul, Div};
+
+use IntoColor;
+use flt;
+use num::{Float, FromPrimitive, Zero};
+use ordered_float::OrderedFloat;
+use white_point::WhitePoint;
+use xyz::Xyz;
+
+/// The smallest wavelength represented in the `Spectrum`
+/// data structure. This is the first value in the
+/// map's wavelength.
+pub const MIN_LAMBDA: usize = 360;
```

Comment:
I had the exact same thought. Will push this change in the next commit, though I was thinking we might also want to include the units? (e.g. MAX_WAVELENGTH_NM).